### PR TITLE
Fix field names

### PR DESCRIPTION
**Pull request by shipperizer**

### CODE REVIEW

File: jira_offline/auth.py [hunk 1]
```diff
@@ -59,9 +59,9 @@ def get_user_creds(project: ProjectMeta, username: Optional[str]=None, password:
         project.password = click.prompt('Password', type=str, hide_input=True)
 
     # validate Jira connection details
-    if project.username and project.password:
```
Comment (mafrosis) [inline jira_offline/auth.py:62]: I'd like to know why this doesn't work rather than commenting it out 😛

**Comment by mafrosis** [comment]
Thanks for the pull request 🗿 

I need to test this, as I suspect the switch from `'name'` to `'displayName'` will break the app for everyone else.

**Comment by shipperizer** [comment]
the `name` key is not there, `emailAddress`, `displayName` and something else can be used.

Re: api HEAD call to `rest/api/2/mypermissions`, have a look at [here](https://community.developer.atlassian.com/t/announcement-changes-to-rest-api-2-mypermissions-get-my-permissions/21245)

The [mypermissions call needs to be a GET](https://developer.atlassian.com/cloud/jira/platform/rest/v2/?utm_source=%2Fcloud%2Fjira%2Fplatform%2Frest%2F&utm_medium=302#api-rest-api-2-mypermissions-get) and the `permissions` query param needs to be one of [these](https://developer.atlassian.com/cloud/jira/platform/rest/v2/?utm_source=%2Fcloud%2Fjira%2Fplatform%2Frest%2F&utm_medium=302#api-rest-api-2-mypermissions-get)

**Comment by mafrosis** [comment]
There is a big difference between Jira API versions and also between Jira Server/Cloud. I expect that is source of the problem with `displayName` vs `name`

**Comment by mafrosis** [comment]
Closed in favour of #73

**Pull request closed without merging**